spec/config/environment_variables_spec.rb

```diff
@@ -13,142 +13,144 @@ def ignore_env_line(config, env)
       "Ignoring \"#{config}\" from environment variable \"#{env}\"."
   end
 
+  MOCK_VIEWER = SpecHelplers::Texdoc::MOCK_VIEWER
```

Comment:
Please use let block instead of defining constant to suppress `warning: already initialized constant MOCK_VIEWER`.

```suggestion
  let(:mock_viewer) { SpecHelplers::Texdoc::MOCK_VIEWER }
```